components/branches/branches.html

```diff
@@ -22,7 +22,11 @@
       </label>
     </div>
     <div class="dropdown-divider"></div>
-    <!-- ko foreach: branchesAndLocalTags -->
+    <div class="form-inline" onclick="event.stopPropagation()">
+      <input class="form-control branch-search-box" type="text" placeholder="Search Branchs" data-bind="value: branchSearchString, valueUpdate: 'afterkeydown'"/>
```

Comment:
```suggestion
      <input class="form-control branch-search-box" type="text" placeholder="Search Branchs" data-bind="value: branchSearchString, valueUpdate: 'afterkeyup'"/>
```

should fix my comment 3.